src/ripple/app/main/Main.cpp

```diff
@@ -169,6 +169,7 @@ printHelp(const po::options_description& desc)
            "     peer_reservations_list\n"
            "     ripple ...\n"
            "     ripple_path_find <json> [<ledger>]\n"
+           "     server_definitions\n"
            "     server_info [counters]\n"
```

Comment:
This is not related to this PR. What does `counters` signify in `server_info [counters]` ?
`server_info` request does not take in any parameters: https://xrpl.org/server_info.html

Reply:
Try it and see.

Reply:
I collected the outputs of `./rippled server_info > a.txt` and `./rippled server_info 2 > b.txt` and compared the diff between the files.

```
7c7
<             "hash" : "BEA5D3D76787DE85172890B884F951FB3C4020138D53D6D380046A2F75DCC780",
---
>             "hash" : "053DB62FB722A8F9F11CD612350C760F4149B98705FADBAB734475B2769A1888",
10c10
<             "seq" : 1885
---
>             "seq" : 1801
13c13
<          "fetch_pack" : 2554,
---
>          "fetch_pack" : 4677,
25c25
<                   "per_second" : 7
---
>                   "per_second" : 3
33c33
<                   "per_second" : 22
---
>                   "per_second" : 20
```

There are some more lines differing in the time statistics, but I'm not able to reason the difference. I couldn't find documentation for `fetch_pack`, but all the other diff'ed fields appear to be consistent with the slightly different start times